File: Códigos em Java/MultiplosDeTresOuCinco.java
/*
 * Se listarmos todos os números naturais abaixo de 10
 * que são múltiplos de 3 ou 5, nós teremos 3, 5, 6 e 9 e
 * a soma desses múltiplos é 23
 * 
 * Problema: Encontre a soma de todos os multiplos de 3 ou 5
 * abaixo de 1000
 */

public class MultiplosDeTresOuCinco {

    public static void main(String[] args) {
        int somar = 0;

        for (int i = 0; i < 1000; i++){
            if (i % 3 == 0 || i % 5 == 0){
                somar += i;
            }
        }

        System.out.println("A soma dos múltiplos de 3 ou 5 abaixo de 1000 é : " + somar);
    }
}

File: Códigos em Python/multiplos_de_tres_ou_cinco.py
"""
Se listarmos todos os números naturais abaixo de 10
que são múltiplos de 3 ou 5, nós teremos 3, 5, 6 e 9 e
a soma desses múltiplos é 23.

Problema: Encontre a soma de todos os múltiplos de 3 ou 5
abaixo de 1000.
"""

somar = sum(x for x in range(1000) if x % 3 == 0 or x % 5 == 0)

print("A soma dos multiplos de 3 e 5 até mil é:", somar)